h2o-docs/src/product/data-science/drf.rst

```diff
@@ -452,6 +452,21 @@ FAQ
 
   -  DRF needs to pass over up to 1M\*22\*250k = 5500 billion numbers per tree, and assuming 50 trees, that’s up to 275 trillion numbers, which can take a few hours
 
+-  **Why does DRF report different training metrics than** ``model_performance`` **?**
+
+  DRF estimates performance of the algorithm on OutOfBound samples (i.e. samples excluded from the sampling mechanism). Since ``sample_rate`` is less than 1 by default, calling ``model_performance`` can give a different number.
+
```

Comment:
OOB means out of bag.

Reply:
Nitpick:

Would love to add your discovery here:

DRF estimates the training metrics performance of the algorithm on OutofBag samples (i.e. samples excluded from the sampling mechanism) and not on the training dataset when sample_rate is less than 1 (by default).  A sample_rate of 1 will generate a warning.

Hence, calling model_performance and set test_data=train will estimate performance on the training dataset and will different from the training metrics.

Reply:
👍